drivers/soundwire/intel_init.c

```diff
@@ -297,6 +307,16 @@ sdw_intel_startup_controller(struct sdw_intel_ctx *ctx)
 		md = link->md;
 
 		md->driver->startup(md);
+
+		if (!link->clock_stop_quirks) {
+			/*
+			 * we need to prevent the parent PCI device
+			 * from entering pm_runtime suspend, so that
+			 * power rails to the SoundWire IP are not
+			 * turned off.
+			 */
+			pm_runtime_get_noresume(link->dev);
```

Comment:
@plbossart this will never happen in our HDA platforms right? Because we always default to BUS_RESET?
This change makes me very uncomfortable for some reason.

Reply:
no, we will need to use quirks == 0 when we need to do acoustic event detection.